### PR TITLE
Update to Spring Boot 1.5.13.RELEASE

### PR DESCRIPTION
**Pull request by gytis**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -88,7 +88,7 @@
     <url>https://github.com/snowdrop</url>
     <connection>scm:git:https://github.com/snowdrop/spring-boot-bom.git</connection>
     <developerConnection>scm:git:git@github.com:snowdrop/spring-boot-bom.git</developerConnection>
-    <tag>1.5.10.Beta1</tag>
+    <tag>1.5.13-SNAPSHOT</tag>
```
Comment (metacosm) [inline pom.xml]: That change doesn't matter as far as I'm aware.

Comment (gytis) [reply pom.xml]: Wouldn't this make a release plugin automatically fix the version?

**Comment by gytis** [comment]
@metacosm updated

File: pom.xml (hunk 1 above)
Comment (metacosm) [reply pom.xml]: I'm not sure how it works, actually but I think the release plugin modifies it when it tags so the value during the SNAPSHOT stage doesn't really matters.

**Comment by metacosm** [comment]
I've actually noted other things that'd need to be changed… 
1. Comments that points to `1.5.11` version (that I should have changed for `1.5.12` already, actually). Not sure what we could do there to avoid having these comments get de-synchronized…
2. Jackson's version is tricky and has been updated. Have you checked the Jackson BOM? Perhaps we shouldn't bother aligning with the specific Jackson BOM that SB uses and align on more recent Jackson version? I'm not sure why the Jackson BOM doesn't use the most recent versions and  if/why SB needs these specific versions…

**Comment by metacosm** [comment]
Otherwise, LGTM.

**Comment by gytis** [comment]
I've actually noted, that we should automate somehow README update as it's quite behind.

Jackson has two issues:

1. Spring Boot is using 2.8.11.20180217 while we use 2.8.11
2. We have Jackson databind as 2.8.11.1 while Jackson BOM imports it using ${project.version}, 2.8.11 in our case.

**Comment by gytis** [comment]
Ah Jackson BOM 2.8.11.20180405 uses databind 2.8.11.1 and jackson 2.8.11 so I guess that's all good.

**Comment by metacosm** [comment]
OK, so you checked that there was no update to `databind` or `annotations` in the new BOM?

**Comment by metacosm** [comment]
Maybe we should use the Jackson BOM as well?

**Comment by gytis** [comment]
By new bom, do you mean 2.8.11.20180405? If so, then `jackson`, `databind` and `annotations` are in line.
Using jackson bom, would make it easier if we would want to start automatically aligning versions with spring-boot-dependencies, but it would bring in extra stuff. So far spring-boot-bom doesn't include any other boms. Do you know if it's just a coincidence or planned decision?

**Comment by gytis** [comment]
Yes of course they won't be pulled in as actual dependencies because they're in dependencyManagement section. But it might increase a possibility of a version overwrite in case different versions are defined in different BOMs. However, in Jackson case that should be ok as it doesn't define third party dependencies.

**Comment by metacosm** [comment]
I think using the Jackson BOM might be the best option to make it easier to automate the dependency update process (when we have one) :)

**Comment by geoand** [comment]
I have created a simple groovy script that attempts to do just that 😀.

It's in one if the branches if I recall

**Comment by gytis** [comment]
OK, so should I add the BOM to this PR or are you just suggesting to do it in the future?

**Comment by metacosm** [comment]
Let's do it as part of the clean-up effort, next time.

**Pull request merged**